internal/msalbase/account_test.go

```diff
@@ -8,17 +8,19 @@ import (
 	"testing"
 )
 
-var accHID = "hid"
-var accEnv = "env"
-var accRealm = "realm"
-var authType = "MSSTS"
-var accLid = "lid"
-var accUser = "user"
+var (
+	accHID   = "hid"
+	accEnv   = "env"
+	accRealm = "realm"
+	authType = "MSSTS"
```

Comment:
>MSSTS [](start = 13, length = 5)

side note: @henrikm we should discuss what MSSTS means...should i open an issue?

Reply:
Yeah, it's just a test value. Has no meaning really AFAIK. MS sts...